packages/dataviews/src/dataviews-layouts/table/style.scss

```diff
@@ -132,6 +132,9 @@
 			display: flex;
 			align-items: center;
 		}
+		.dataviews-view-table__cell-content-wrapper:empty {
```

Comment:
@jameskoster How should I test this is working properly? I couldn't do a proper git blame to understand where this rule comes from due to files being moved around, so I presume you'd know something about this :)

Reply:
I'm afraid I don't exactly remember. I agree it doesn't seem to serve a purpose which suggest it relates to part of the UI that has evolved over time. 

Since it hasn't been working for a while, and doesn't seem to affect anything maybe it's safe to remove?

Reply:
Yeah, it makes more sense. Pushed the removal.